.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,46 @@
+name: "Coding Standards"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
```

Comment:
I'm assuming this is a matrix on purpose even though it will run on just a single version.

Reply:
Hmm good point, not sure why we (I, probably) went that way… maybe the goal was to be able to reuse that variable in different steps?

Reply:
I don't mind if we keep the PHP version in a matrix.